File: backend/dataset/levelOrder/levelorder_6.py
class Solution:
    def levelOrder(self, root: TreeNode) -> List[List[int]]:
        if root is None:
            return root
        
        queue=[]
        
        queue.append(root)
        ans=[]
        while queue:
            l=len(queue)
            level=[]
            while l!=0:
                node= queue.pop(0)
                level.append(node.val)
            
                if node.left:
                    queue.append(node.left)
                if node.right:
                    queue.append(node.right)
                l=l-1
            ans.append(level)     
                
        return ans             

File: backend/CDataset/levelOrder/levelOrder_6.cpp
class Solution {
public:

    class TreeNode{
        public:
        TreeNode* left;
        TreeNode* right;
        int val;
    };

void bfs(TreeNode*root , vector<int>&v,vector<vector<int>>&qwer){
queue<TreeNode*>q;
if(root==NULL){
return;}

q.push(root);
q.push(NULL);

while(!q.empty()){
TreeNode*f=q.front();

if(f==NULL){
qwer.push_back(v);
    v.clear();
q.pop();
if(!q.empty()){
q.push(NULL);}}

else{
v.push_back(f->val);
q.pop();
 if(f->left){
q.push(f->left);}
if(f->right){
q.push(f->right);}}}
    return ;}
	
	
	vector<vector<int>> levelOrder(TreeNode* root) {
     vector<vector<int>>qwer;
    vector<int>v;
    bfs(root,v,qwer);
     return qwer;}
};
